src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -895,6 +926,7 @@ private TrackBitrateAllocation(
             // consider 720p@30fps, 360p@30fps, 180p@30fps, 180p@15fps,
             // 180p@7.5fps while for the thumbnails we consider 180p@30fps,
             // 180p@15fps and 180p@7.5fps
+            int preferredIdx = -1;
```

Comment:
because the instance field is final and we update the local variable multiple times inside the loop